### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

**What this PR does / why we need it**:

We need to release new manifest templates.

**Special notes for your reviewer**:

### CODE REVIEW

File: version/description [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-* dp: Expose link based resources by default (#11)
-* Unify template manifest (#20)
+* Adding quotes in macvtap.yaml.in, namespace.yaml.in,scc.yaml.in (#24)
+
+Bugs:
+* Change vbom.ml/util dependeny with github.com/fvbommel/util (#25)
+
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.2.0
+docker pull HANDLER_IMAGE
```
Comment (maiqueb) [inline version/description:10]: Please put here the correct image name.

**Review by maiqueb — CHANGES_REQUESTED** [review]
Please use the correct img name in the `version/description` file.

File: version/description (hunk 1 above)
Comment (phoracek) [reply version/description:10]: I had a feeling that this is replaced by our release script, but I don't see it being implemented. Fixing.

**Review by maiqueb — APPROVED** [review]
/lgtm
/approve

Thanks.

**Pull request merged**